src/reducers/notesReducer.jsx

```diff
@@ -1,44 +1,90 @@
 const InitialValues = {
-    notesList: [],
-    archiveList: [],
-    trashList: [], 
-    isEditing:false,
-    editNote:{title:"", note:"",tags:[]}
-  };
-  
-  const NotesReducerFun = (notesState, { type, payload }) => {
-    switch (type) {
-      case "GET_NOTES":
-        return { ...notesState, notesList: payload.notesList };
-      case "SET_NOTES":
-        return { ...notesState, notesList: payload.notesList };
-      case "GET_ARCHIVE_NOTES":
-        return {
-          ...notesState,
-          notesList: payload.notesList,
-          archiveList: payload.archiveList,
-        };
-      case "SET_ARCHIVE_NOTES":
+  notesList: [],
+  archiveList: [],
+  trashList: [],
+  isEditing: false,
+  editNote: { title: "", note: "", tags: [] },
+  sortByDate: "",
+  sortByPriority: "",
+  priority: {
+    HIGH: false,
+    MEDIUM: false,
+    LOW: false,
+  },
+};
+
+const NotesReducerFun = (notesState, { type, payload }) => {
+  switch (type) {
+    case "GET_NOTES":
+      return { ...notesState, notesList: payload.notesList };
+    case "SET_NOTES":
+      return { ...notesState, notesList: payload.notesList };
+    case "GET_ARCHIVE_NOTES":
```

Comment:
Suggestion:  You can convert these action types into constant and store them in a separate file.